src/libraries/Microsoft.Extensions.Configuration.Binder/src/ConfigurationBinder.cs

```diff
@@ -698,6 +746,37 @@ private static Array BindArray(Array source, IConfiguration config, BinderOption
                 options);
         }
 
+        // todo: steve - we might not need this; currently, these attributes are only applicable to properties and
```

Comment:
What's the plan for addressing this?

I agree that this code doesn't make sense because these attributes can't be on parameters:

https://github.com/dotnet/runtime/blob/bd43f55230d533af326e727366f61cfd3ed0e050/src/libraries/Microsoft.Extensions.Configuration.Abstractions/src/ConfigurationKeyNameAttribute.cs#L8-L9